jlm-hls/include/jlm-hls/cmdline.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 Magnus Sjalander <work@sjalander.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#ifndef JLM_JLMHLS_CMDLINE_HPP
+#define JLM_JLMHLS_CMDLINE_HPP
+
+#include <jlm/util/file.hpp>
+#include <jlm/util/Statistics.hpp>
+
+#include <string>
+#include <vector>
```

Comment:
Is the "#include <vector>" really needed? I do not see a std::vector used in this file?

Reply:
string and vector have been removed